file_formats/ie_formats/are_v1.htm

```diff
@@ -97,24 +97,48 @@
         <tr>
           <td>0x0014</td>
           <td>4 (dword)</td>
-          <td>Area flag (AREAFLAG.IDS)<br />
+          <td>Area flags (AREAFLAG.IDS)<br />
             <br />
             BG1:TotS, IWD:ToTL, BG2:ToB:
             <ul>
-              <li>bit 0: Save allowed</li>
+              <li>bit 0: Save not allowed</li>
               <li>bit 1: Tutorial area (not BG1)</li>
               <li>bit 2: Dead magic zone</li>
               <li>bit 3: Dream</li>
             </ul>
 
+            BGEE:
+            <ul>
+              <li>bit 0: Save not allowed</li>
+              <li>bit 1: Tutorial area</li>
+              <li>bit 2: Dead magic zone</li>
+              <li>bit 3: Dream</li>
+              <li>bit 4: Player1 death does not end the game</li>
+              <li>bit 5: Resting not allowed</li>
+              <li>bit 6: Travel not allowed</li>
+            </ul>
+
             PST:
             <ul>
-              <li>0 - Rest allowed</li>
-              <li>2 - "You cannot rest here."</li>
-              <li>4 - "You cannot rest right now."</li>
-              <li>6 - "You must obtain permission to rest here."</li>
+              <li>bit 0: Save not allowed</li>

```

Comment:
Are you sure? I don't this bit is used in pst.

Reply:
The bit seems to work as described in PST. I have just tested it to make sure.

Reply:
ok, good to know. Our previous research is wrong then - maybe it hinged on the old iesdp description.